File: src/main/java/com/distribution/modules/dis/vo/CardOrderInfoVO.java
package com.distribution.modules.dis.vo;

import lombok.Data;

@Data
public class CardOrderInfoVO {
    /**
     *
     */
    private String orderName;
    /**
     *
     */
    private String orderId;
    /**
     *
     */
    private String orderMobile;
    /**
     *
     */
    private String orderIdcardno;
    /**
     *
     */
    private String orderEmail;
    /**
     * 信用卡ID
     */
    private String cardId;
    /**
     * 订单状态 0:失败,1:成功,2:申请中
     */
    private Integer orderStatus;


    public static final class OrderStatus {
        public static final int LOSE = 0;//失败
        public static final int SUCCESS = 1;//成功
        public static final int APPLICATION = 2;//申请中
    }
}

File: src/main/kotlin/com/distribution/modules/dis/vo/CardOrderInfoVO.kt
package com.distribution.modules.dis.vo


data class CardOrderInfoVO(
        /**
         * 真实姓名
         */
        var orderName: String = "",

        /**
         *
         */
        var orderId: String = "",

        /**
         * 手机号
         */
        var orderMobile: String = "",

        /**
         * 身份证
         */
        var orderIdcardno: String = "",

        /**
         *
         */
        var orderEmail: String = "",

        /**
         * 信用卡编号
         */
        var cardNum: String = "",

        /**
         * 银行编号
         */
        var bankNum: String = "",

        /**
         * 订单状态 0:失败,1:成功,2:申请中
         */
        var orderStatus: Int = 0
) {
    object OrderStatus {
        val LOSE = 0//失败
        val SUCCESS = 1//成功
        val APPLICATION = 2//申请中
    }
}
